lib/message.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
```

Comment:
Everything looks great here! The next iteration could involve trying to reduce some of the duplication.